.github/workflows/protocol-deep-invarianats.yaml

```
@@ -158,8 +158,8 @@ jobs:
           FOUNDRY_FUZZ_RUNS: ${{ inputs.fuzzRuns || '2000' }}
           MAINNET_RPC_URL: ${{ secrets.TESTS_MAINNET_RPC_URL }}
           ARBITRUM_ONE_RPC_URL: ${{ secrets.TESTS_ARBITRUM_ONE_RPC_URL }}
-  
+
       - name: "Add test summary"
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:42: Double quote to prevent globbing and word splitting

(shellcheck)